### PR TITLE
Add branding

### PR DESCRIPTION
**Pull request by awisniew90**
Screen Shot 2023-03-17 at 11 19 37 AM
Screen Shot 2023-03-17 at 11 19 56 AM

Fixes https://github.com/OpenLiberty/liberty-tools-eclipse/issues/195

### CODE REVIEW

File: bundles/io.openliberty.tools.eclipse.product/about.properties [hunk 1]
```diff
@@ -0,0 +1,6 @@
+blurb=Liberty Tools for Eclipse IDE\n\
+\n\
+Version: {featureVersion}\n\
+\n\
+Copyright (c) 2023 IBM Corporation and others.\n\
```
Comment (scottkurz) [inline bundles/io.openliberty.tools.eclipse.product/about.properties:5]:
```suggestion
Copyright (c) 2022, 2023 IBM Corporation and others.\n\
```

Though this part is new, this is describing the feature as a whole, so I guess the <initial,latest> format is better?

File: bundles/io.openliberty.tools.eclipse.product/about.properties [hunk 2]
```diff
@@ -0,0 +1,6 @@
+blurb=Liberty Tools for Eclipse IDE\n\
+\n\
+Version: {featureVersion}\n\
```
Comment (scottkurz) [inline bundles/io.openliberty.tools.eclipse.product/about.properties:3]: No issue...I'm just curious how you found this built-in property 'featureVersion'?   In the doc or just from some other project?

**Review by scottkurz — APPROVED** [review]

File: bundles/io.openliberty.tools.eclipse.product/META-INF/MANIFEST.MF [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Manifest-Version: 1.0
+Bundle-Copyright: Copyright (c) 2023 IBM Corporation and others.
+Bundle-ManifestVersion: 2
+Bundle-Name: Liberty Tools UI
```
Comment (scottkurz) [inline bundles/io.openliberty.tools.eclipse.product/META-INF/MANIFEST.MF:4]:
```suggestion
Bundle-Name: Liberty Tools
```

**Review by scottkurz — CHANGES_REQUESTED** [review]

File: bundles/io.openliberty.tools.eclipse.product/about.properties (hunk 2 above)
Comment (awisniew90) [reply bundles/io.openliberty.tools.eclipse.product/about.properties:3]: I found it in some example Eclipse projects.

**Review by scottkurz — APPROVED** [review]

**Pull request merged**